pipelines/common/src/test/resources/sql-on-fhir-v2-tests/fn_reference_keys.json

```diff
@@ -59,20 +64,25 @@
           {
             "column": [
               {
-                "path": "getResourceKey() = link.other.getReferenceKey(Patient)",
-                "name": "key_equal_ref",
-                "type": "boolean"
+                "path": "getResourceKey()",
+                "name": "resourceKey"
+              },
+              {
+                "path": "link.other.getReferenceKey('Patient')",
```

Comment:
Yeah, it was not clear to me either since there are both examples with and without `'`. I took another look and I think in most cases it is without `'`, e.g., `getReferenceKey(Patient)`. It is also consistent with the FHIRPath [`ofType` function](https://hl7.org/fhirpath/#oftypetype-type-specifier-collection). So I went ahead and change everywhere to no `'`. I'll send a PR to update the specification and remove cases with `'` later.

Reply:
Got it and thanks for the changes.